proto/sequencers/tx.proto

```diff
@@ -1,43 +1,40 @@
 syntax = "proto3";
 package rollapp.sequencers.types;
 
-import "gogoproto/gogo.proto";
 import "cosmos/msg/v1/msg.proto";
 import "google/protobuf/any.proto";
 
 option go_package = "github.com/dymensionxyz/dymension-rdk/x/sequencers/types";
 
 // Msg defines the Msg service.
 service Msg {
-    rpc CreateSequencer(MsgCreateSequencer) returns (MsgCreateSequencerResponse);
-    rpc UpdateSequencer(MsgUpdateSequencer) returns (MsgUpdateSequencerResponse);
+    // Messages
+    rpc UpdateRewardAddress(MsgUpdateRewardAddress) returns (MsgUpdateRewardAddressResponse);
+    rpc UpdateWhitelistedRelayers(MsgUpdateWhitelistedRelayers) returns (MsgUpdateWhitelistedRelayersResponse);
 
+    // Consensus Messages
     rpc UpsertSequencer(ConsensusMsgUpsertSequencer) returns (ConsensusMsgUpsertSequencerResponse);
```

Comment:
how this TX prevented from being sent manually on the L2?

Reply:
as discussed, it can be sent by only by the owner of the address (i.e sequencer can update it's own reward and relayer address which is fine)

Reply:
I'm not talking about the update msgs, but the `UpsertSequencer` which creates new sequencer.
It's supposed to be treated solely as Consesnsus msg

Reply:
anyway, if it can be run by L2 users, we need to validate the pubkey (as we had in `create_sequencer` before).
I prefer to handle it as trusted consensus msg

Reply:
as discussed gonna block it from running by the L2.